tests/components/mystrom/test_init.py

```diff
@@ -0,0 +1,139 @@
+"""Test the myStrom init."""
+from unittest.mock import AsyncMock, PropertyMock, patch
+
+from pymystrom.exceptions import MyStromConnectionError
+
+from homeassistant.config_entries import ConfigEntryState
+from homeassistant.core import HomeAssistant
+
+from .conftest import DEVICE_NAME
+
+from tests.common import MockConfigEntry
+
+DEVICE_MAC = "6001940376EB"
+
+
+async def init_integration(
+    hass: HomeAssistant,
+    config_entry: MockConfigEntry,
+    device_type: int,
+    bulb_type: str = "strip",
+) -> None:
+    """Inititialize integration for testing."""
+    with patch(
+        "pymystrom.get_device_info",
+        side_effect=AsyncMock(return_value={"type": device_type, "mac": DEVICE_MAC}),
+    ), patch("pymystrom.switch.MyStromSwitch.get_state", return_value={}), patch(
+        "pymystrom.bulb.MyStromBulb.get_state", return_value={}
+    ), patch(
+        "pymystrom.bulb.MyStromBulb.bulb_type", bulb_type
+    ), patch(
+        "pymystrom.switch.MyStromSwitch.mac",
+        new_callable=PropertyMock,
+        return_value=DEVICE_MAC,
+    ), patch(
+        "pymystrom.bulb.MyStromBulb.mac",
+        new_callable=PropertyMock,
+        return_value=DEVICE_MAC,
+    ):
+        await hass.config_entries.async_setup(config_entry.entry_id)
+        await hass.async_block_till_done()
+
+    assert config_entry.state == ConfigEntryState.LOADED
+
+
+async def test_init_switch_and_unload(
+    hass: HomeAssistant, config_entry: MockConfigEntry
+) -> None:
+    """Test the initialization of a myStrom switch."""
+    await init_integration(hass, config_entry, 101)
+    state = hass.states.get("switch." + DEVICE_NAME)
+    assert state is not None
+    await hass.config_entries.async_unload(config_entry.entry_id)
+    await hass.async_block_till_done()
+    state = hass.states.get("switch.test")
+    assert state is None
+
+
+async def test_init_bulb(hass: HomeAssistant, config_entry: MockConfigEntry) -> None:
+    """Test the initialization of a myStrom bulb."""
+    await init_integration(hass, config_entry, 102)
+    state = hass.states.get("light." + DEVICE_NAME)
```

Comment:
I suggest that you hard-code it here.
```suggestion
    state = hass.states.get("light.xxx")
```

Reply:
done